src/processor/mapper/map_expressions_scan.cpp

```diff
@@ -18,7 +18,9 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapLogicalExpressionsScanToPhysica
     auto sharedState = std::make_shared<FTableSharedState>();
     // populate static table
     std::unique_ptr<FactorizedTableSchema> tableSchema = std::make_unique<FactorizedTableSchema>();
+    // TODO(Ziyi): remove vectors when we have done the refactor of dataChunk.
     std::vector<std::shared_ptr<ValueVector>> vectors;
```

Comment:
Can't u remove vectors?

Reply:
No, vectors need to take ownership of the valueVectors because the expression evaluator will be destroyed after.